@here/harp-mapview/lib/MapView.ts

```diff
@@ -867,6 +868,7 @@ export class MapView extends THREE.EventDispatcher {
         this.m_camera.up.set(0, 0, 1);
         this.m_lookAtDistance = 0;
         this.m_focalLength = 0;
+        this.m_maxVisibility = 4000000;
```

Comment:
does this mean anything specifically? Would it be better to use MAX_SAFE_INTEGER?

Reply:
This is exactly the far plane distance setup initially for camera, before the cameras and clip planes evaluator update. See few lines above. I guess MAX_SAFE_INTEGER is not the best option for frustum settings and this need to be compliant - the same.

Reply:
Now I see, ok, great you made a constant.